README.md

```diff
@@ -231,9 +233,11 @@ The most recent matching source is given a `credit` of value 100. All other matc
 
 For each matching source, schedule a report. To schedule a report,
 the browser will store
- {`attributionreportto`, `attributiondestination` eTLD+1, `attributionsourceeventid`, [decoded](#data-encoding) trigger-data, credit} for the source.
+ {`attributionreportto`, `attributiondestination` eTLD+1, `attributionsourceeventid`, [decoded](#data-encoding) trigger-data, credit, dedup-key} for the source.
 Scheduled reports will be sent as detailed in [Sending scheduled reports](#sending-scheduled-reports).
 
+The browser will only create reports for a source if the trigger's dedup-key has not already been associated with a report for that source.
```

Comment:
Does this imply that the browser stores every dedup-key value for every source forever? Or is the dedup-key only valid for a browser-determined period?

Reply:
The dedup-keys can only deduplicate triggers for the source they are attributed to. The browser only needs to keep it around for the lifetime of the source.

So for example, if we have:

source A
trigger 1, dedup=1
trigger 2, dedup=1
source B
trigger 3, dedup=1

The browser will send 2 reports:

<source A, trigger 1>
<source B, trigger 3>